build/tasks/validate.js

```diff
@@ -307,5 +301,170 @@ function validateRule({ tags, metadata }) {
   if (help.toLowerCase().includes(prohibitedWord)) {
     issues.push(`metadata.help can not contain the word '${prohibitedWord}'.`);
   }
+
+  issues.push(...findTagIssues(tags));
   return issues;
 }
+
+const uniqueTags = ['ACT', 'experimental', 'review-item', 'deprecated'];
+
+const categories = [
+  'aria',
+  'color',
+  'forms',
+  'keyboard',
+  'language',
+  'name-role-value',
+  'parsing',
+  'semantics',
+  'sensory-and-visual-cues',
+  'structure',
+  'tables',
+  'text-alternatives',
+  'time-and-media'
+];
+
+const standardsTags = [
+  {
+    name: 'WCAG',
+    standardRegex: /^wcag2(1|2)?a{1,3}$/,
+    criterionRegex: /^wcag\d{3,4}$/
+  },
+  {
+    name: 'Section 508',
+    standardRegex: /^section508$/,
+    criterionRegex: /^section508\.\d{1,2}\.[a-z]$/,
+    wcagLevelRegex: /wcag2aa?/
```

Comment:
Should the `wcagLevelRegex`es be terminated by `^` and `$` to avoid, say, matching `wcag2aaa` by accident?

Reply:
Yes it should :) Fixed